scout/server/blueprints/variants/forms.py

```diff
@@ -123,6 +123,8 @@ class VariantFiltersForm(FlaskForm):
     filter_variants = SubmitField(label="Filter variants")
     export = SubmitField(label="Filter and export")
 
+    hide_unaffected = BooleanField("Hide variants present only in unaffected", default="checked")
```

Comment:
Ok I found out that if I use render_kw={'checked': True} instead of default="checked", then it works.

Other people with the same problem: https://github.com/wtforms/flask-wtf/issues/362

Reply:
But then if you uncheck and submit the form it doesn't stay unchecked when the page reloads :expressionless:

Reply:
How about the js suggestion they had on the page you linked, like setting the checked if the value is true?

Reply:
I'm trying with jinja

Reply:
Didn't work, tried also with jquery. So I made it like this: 😆 

<img width="273" alt="image" src="https://user-images.githubusercontent.com/28093618/156587447-0f40f872-4bd8-434d-a16f-8afbb758e304.png">